docs/Format.md

```diff
@@ -131,7 +131,7 @@ text for all fields, starting at column 25. Remove fields that do not apply.
 ```
 
 
-______________________________________________________________________________
+________________________________
```

Comment:
```suggestion
________
```
The idea here is to make it equal in length to the first metadata's line (including values). More of a visual thing (neither too short nor dangles at the end). I'll soon make a non-failing test for that as well.